README.md

```diff
@@ -1,172 +1,48 @@
-<a name="readme-top"></a>
-</div>
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-<!-- TABLE OF CONTENTS -->
+# Rails Blog App
 
-# 📗 Table of Contents
+> The Blog app is Roby on rails website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
 
-- [📗 Table of Contents](#-table-of-contents)
-- [📖 \[Ruby Capstone\] ](#-ruby-capstone-)
-  - [🛠 Built With ](#-built-with-)
-    - [Tech Stack ](#tech-stack-)
-    - [Key Features ](#key-features-)
-  - [💻 Getting Started ](#-getting-started-)
-    - [Prerequisites](#prerequisites)
-    - [Setup](#setup)
-    - [Install](#install)
-    - [Usage](#usage)
-    - [Run tests](#run-tests)
-  - [👥 Authors ](#-authors-)
-  - [🔭 Future Features ](#-future-features-)
-  - [🤝 Contributing ](#-contributing-)
-  - [⭐️ Show your support ](#️-show-your-support-)
-  - [🙏 Acknowledgments ](#-acknowledgments-)
-  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
-  - [📝 License ](#-license-)
 
-<!-- PROJECT DESCRIPTION -->
+## Built With
+- Languages: Ruby
+- Models/Frameworks: Ruby on Rails
 
-# 📖 [Blog App] <a name="about-project"></a>
+## Getting started
+To get a local copy of this exercice, Please follow these simple example steps.
 
-**[Blog App]** Blog App is a fully functional Ruby on Rails application that will show the list of posts and empower users to interact by adding comments and liking posts.
+#### 1. Clone this repository or download the Zip folder:"
```

Comment:
- [ ] I see that you have the correct updated Readme file on your default dev branch, I suggest you to use that same Readme file, as this one is not valid, please do update your Readme in this branch :+1:

Reply:
Thanks, i noticed i re add the old one.